### PR TITLE
Fix lang

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

File: build/lang/ar.js [hunk 1]
```diff
@@ -11,21 +11,21 @@ window.D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior = window.D2L.P
...
 		'bigTrendAttemptsScreenReaderString': '{numAttempts, plural, =1 {Attempt {attemptNames}} other {Attempts {attemptNames} and {lastAttemptName}}}',
-		'bigTrendAttemptsTooltipString': '{numAttempts, plural, =1 {محاولة} other { من المحاولات}} {attemptNames}',
-		'close': 'إغلاق',
-		'evidence': 'دليل',
-		'headingDate': 'التاريخ',
-		'headingEvidence': 'اسم الدليل',
-		'headingLoa': 'مستوى التحصيل',
-		'miniTrendAttemptsTooltipString': '{numAttempts, plural, =1 {محاولة} other { من المحاولات}} {attemptNames}',
-		'miniTrendScreenReaderText': 'Assessed {numAssessed} {numAssessed, plural, =1 {مرة} other {من المرات}}: {levelNames}',
+		'bigTrendAttemptsTooltipString': '{numAttempts, plural, =1 {Attempt} other {Attempts}} {attemptNames}',
```
Comment (sarahboyd) [inline build/lang/ar.js]: should these have been reverted too?

Comment (rgoessl) [reply build/lang/ar.js]: fixed

File: build/lang/tr.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior.LangTrBehavior = {
 		'headingEvidence': 'Kanıt Adı',
 		'headingLoa': 'Başarı Seviyesi',
 		'miniTrendAttemptsTooltipString': '{numAttempts, plural, =1 {Deneme} other {Deneme}} {attemptNames}',
-		'miniTrendScreenReaderText': '{numAssessed} {numAssessed, plural, =1 {kez} other {kez}} değerlendirildi: {levelNames}',
+		'miniTrendScreenReaderText': 'Assessed {numAssessed} {numAssessed, plural, =1 {kez} other {kez}}: {levelNames}',
```
Comment (sarahboyd) [inline build/lang/tr.js]: this one still needs to be reverted

Comment (rgoessl) [inline build/lang/tr.js]:
```suggestion
		'miniTrendScreenReaderText': '{numAssessed} {numAssessed, plural, =1 {kez} other {kez}} değerlendirildi: {levelNames}',
```

**Pull request merged**